client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
Oh wait, sorry. That's not quite right. I misread the first prompt. The prompt for the first thing would be:

">>> x=5\n>>> square(x)\nhash(25)\n>>>square(-x)\nhash(25)"

The prompt for the second thing is correct. And yeah, this can be implemented easily since we have access to the full text on each unlocking answer.